### PR TITLE
Schedule C - Expected Use and Other column

### PR DESCRIPTION
**Pull request by amichard**
#1151 #1152 #1154

Added options for Expected Use and conditions for Other column

Changelog:
- Added model and migration for Expected Uses
- Added viewset, serializer for Expected Uses
- Loading options for Expected Use in the spreadsheet
- Added a trigger to enable/disable the last column as needed
- Added row number
- Changed hard-coded column number to use constants instead
- Added totals as a floating object on the lower right portion of the page
- Allow adding multiple rows

### CODE REVIEW

File: backend/api/models/ExpectedUse.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        unique=True,
+        db_comment="Description on what the fuel is going to be used for."
+    )
+
+    objects = ExpectedUseManager()
+
+    class Meta:
+        db_table = 'expected_use'
+
+    db_table_comment = "Different options for the Expected Use column in " \
```
Comment (klefler) [inline backend/api/models/ExpectedUse.py]: Is it possible to expand the description? It's not clear what "expected use" is... and maybe clarify the name of the "Schedule C" report? Or may not even need to specify in the description the specific use for that schedule... if there's a chance this code table could be used for other things.

File: backend/api/models/ExpectedUse.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    description = models.CharField(
+        max_length=100,
+        unique=True,
+        db_comment="Description on what the fuel is going to be used for."
+    )
+
+    objects = ExpectedUseManager()
+
+    class Meta:
+        db_table = 'expected_use'
```
Comment (klefler) [inline backend/api/models/ExpectedUse.py]: If this is a lookup table, should the table name be similar to others? i.e. "expected_use_type"

Comment (amichard) [reply backend/api/models/ExpectedUse.py]: sure i can name it to expected_use_type

File: backend/api/models/ExpectedUse.py (hunk 1 above)
Comment (amichard) [reply backend/api/models/ExpectedUse.py]: ill try to provide a better explanation on what the table does

**Comment by amichard** [comment]
[Compliance Report C | Show the total fuel excluded for each fuel class |2|](https://trello.com/c/OdveyJjN/1303-compliance-report-c-show-the-total-fuel-excluded-for-each-fuel-class-2)

**Comment by amichard** [comment]
[Compliance Report C | An additional column appears when Other is selected for Expected Use |2|](https://trello.com/c/eUq1jomb/1302-compliance-report-c-an-additional-column-appears-when-other-is-selected-for-expected-use-2)

**Comment by amichard** [comment]
[Compliance Report C | Provide predefined selections for Expected Use column |3|](https://trello.com/c/cCaMiWSZ/1301-compliance-report-c-provide-predefined-selections-for-expected-use-column-3)

**Pull request merged**